### DISCUSSION TITLE
Show Item in Hover State when editing the hover styling (Migrated)

### DISCUSSION THREAD

**Discussion by diickens** [elementor/elementor#30212] [Feature Request]
When edting a button or any other item with a hover state, it would great when the element would switch in to hover state when editing the hover style of it.

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #10534  

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Don't worry, your comments, reactions, and participation in the old issue will be accounted for when considering this Feature Request for our Roadmap.

Happy voting! 🙂
